### PR TITLE
Ben

### PR DESCRIPTION
**Pull request by BenMiddleton**
Unit testing, cleanup, styles etc.

### CODE REVIEW

File: Coop_Listing_Site/Coop_Listing_Site/Content/control-panel-layout.css [hunk 1]
```diff
@@ -1,9 +1,10 @@
 ﻿.fadeout {
```
Comment (Chef-Code) [inline Coop_Listing_Site/Coop_Listing_Site/Content/control-panel-layout.css]: I end up erasing this all together in my pull request, we don't use it

File: Coop_Listing_Site/Coop_Listing_Site/Content/control-panel-layout.css [hunk 2]
```diff
@@ -1,17 +0,0 @@
-﻿.fadeout {
```
Comment (Chef-Code) [inline Coop_Listing_Site/Coop_Listing_Site/Content/control-panel-layout.css:1]: @BenMiddleton OK, good.  My thoughts exactly... we don't need this

**Comment by Chef-Code** [comment]
@BenMiddleton @Kethsar @droopy25 @gafler 
This looks good, I think we should accept my pull request first although... Because my pull request will try to reintroduce all of the deleted views and methods proposed in this pull request.  What do you guys think??

**Comment by Kethsar** [comment]
Indeed looks fine, but I will not do your first cause I do have some questions about it, @Chef-Code

**Pull request merged**